temba/api/v2/views.py

```diff
@@ -3573,3 +3581,454 @@ def post(self, request, *args, **kwargs):
+
+            PUT /api/v2/database_records.json
+            {
+                "collection_name": "Lookups List",
+                "objectId": "J7vDtb5Aek",
+                "item": {
+                    "name": "Gendalf the Gray"
+                }
+            }
+
+
+        Response status of success updating would be `202`:
+
+            {
+                "updatedAt": "2020-12-10T20:24:32.595Z"
+            }
+
+        ## Lookup Record Deleting
+
+        A `DELETE` deletes collection:
+
+         * **collection_name** - the name of collection
+         * **objectId** - identifier of record to delete
+
+        Create new collection for current org:
+
+            DELETE /api/v2/database_records.json
+            {
+                "collection_name": "Lookups List",
+                "objectId": "J7vDtb5Aek"
+            }
+
+        Response status of success deletion would be `204`
+    """
+
+    permission = "orgs.org_lookups"
+
+    def list(self, request, *args, **kwargs):
+        org, collection_name, collections_list, error_response = self.get_default_params(config="collection_exist")
+        if error_response:
+            return error_response
+
+        collection = self.get_collection_full_name(org=org, collection=collection_name)
+        parse_headers = {
+            "X-Parse-Application-Id": settings.PARSE_APP_ID,
+            "X-Parse-Master-Key": settings.PARSE_MASTER_KEY,
+            "Content-Type": "application/json",
+        }
+        results_url = f"{settings.PARSE_URL}/classes/{collection}?order=order&limit=1000"
+        response = requests.get(results_url, headers=parse_headers)
+        result = response.json()
+
+        return Response(result, status=status.HTTP_200_OK)
+
+    def post(self, request, *args, **kwargs):
+        org, collection_name, collections_list, error_response = self.get_default_params(config="collection_exist")
+        if error_response:
+            return error_response
+
+        items_to_push = self.request.data.get("items", [])
+        if not items_to_push:
+            return Response(status=status.HTTP_400_BAD_REQUEST)
+
+        collection = self.get_collection_full_name(org=org, collection=collection_name)
+        count_url = f"{settings.PARSE_URL}/classes/{collection}?count=1"
+        count_response = requests.get(count_url, headers=self.parse_headers)
+
+        if count_response.status_code == 200:
+            requests_ = []
+            insert_url = f"{settings.PARSE_URL}/batch"
+            insert_index = count_response.json().get("count")
+            db_endpoint = f"{settings.PARSE_ENDPOINT}/classes/{collection}"
+            for index, data in enumerate(items_to_push, start=insert_index):
+                requests_.append({"method": "POST", "path": db_endpoint, "body": {"order": index, **data}})
+            response = requests.post(insert_url, data=json.dumps({"requests": requests_}), headers=self.parse_headers)
```

Comment:
See the comment on https://github.com/greatnonprofits-nfp/rapidpro/pull/170/files#diff-cfdc7e7ca24015478ba706e07f1a13347724538d0fdbec177333860d7e4d8b37R3834

Reply:
Added batching to create request